connection.go

```diff
@@ -510,7 +510,28 @@ func (v *connection) authSendSHA512Password(extraAuthData []byte) error {
 }
 
 func (v *connection) sync() error {
-	return v.sendMessage(&msgs.FESyncMsg{})
+	err := v.sendMessage(&msgs.FESyncMsg{})
+
+	if err != nil {
+		return err
+	}
+
+	for true {
+		bem, err := v.recvMessage()
+		if err != nil {
+			return err
+		}
+
+		_, ok := bem.(*msgs.BEReadyForQueryMsg)
+
+		if ok {
+			break
+		}
+
+		_, _ = v.defaultMessageHandler(bem)
```

Comment:
If you're ignoring both return values you can drop the `_, _ =` half of this line. Is there any danger in ignoring the error here?

Reply:
No danger in ignoring them. I often drop the explicit ignored returns, but GoLand claims that it's proper practice to be explicit. I let GoLand fix it by itself and really like showing that I'm aware of ignoring them.

Reply:
If a linter is recommending it then fair enough, best to have consistency through the code base.